src/index.js

```diff
@@ -8,13 +8,15 @@ import { Reducers } from './reducers';
 import { Actions } from './actions';
 
 import { applyGridConfig } from './constants/GridConstants';
+import * as ActionTypes from './constants/ActionTypes';
```

Comment:
Just delete the `GridRootReducer: combineReducers(Reducers),` since we still need to export the Reducers.